### PR TITLE
space and block creation

### PR DESCRIPTION
**Pull request by sajdakabir**

### CODE REVIEW

File: apps/frontend/src/components/TodayMeetings.tsx [hunk 1]
```diff
@@ -55,7 +60,7 @@ export const TodayMeetings: React.FC<TodayAgendaProps> = ({ selectedDate }) => {
           <React.Fragment key={index}>
             <li className="text-[#DCDCDD]/80 text-lg font-medium">{item.title}</li>
             <p>{item.time}, {item.duration} min</p>
-            <a href={item.link} className="text-[#DCDCDD] mt-4 mb-8 flex justify-start items-center gap-2">
+            <a href={item.link} target="_blank" className="text-[#DCDCDD] mt-4 mb-8 flex justify-start items-center gap-2">
```
Comment (madhav-relish) [inline apps/frontend/src/components/TodayMeetings.tsx:60]: @deewakar-k  Can you use Link tag from next.js here.

File: apps/frontend/src/components/TodayTextArea.tsx [hunk 2]
```diff
@@ -31,26 +31,21 @@ export const TodayTextArea = ({ selectedDate }: JournalProps): JSX.Element => {
   });
 
   useEffect(() => {
-    console.log('Fetching journal for date: ', formattedDate);
+    console.log("Fetching journal for date: ", formattedDate);
```
Comment (madhav-relish) [inline apps/frontend/src/components/TodayTextArea.tsx]: Remove this, and other console.log

Comment (deewakar-k) [reply apps/frontend/src/components/TodayTextArea.tsx]: done!

File: apps/frontend/src/components/TodayMeetings.tsx (hunk 1 above)
Comment (deewakar-k) [reply apps/frontend/src/components/TodayMeetings.tsx:60]: Screenshot From 2024-10-03 16-49-00
{item.link} is external link that's why i choosed anchor over Link im not doing any client side routing. @madhav-relish

**Pull request merged**